<launch>
	<arg name="ugv_id" default="1"/>
	<node pkg="prometheus_ugv_control" type="ugv_terminal_control" name="ugv_terminal_control_$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
	</node>
</launch>

